Hakawai/Mentions/HKWMentionsPluginV2.m

```diff
@@ -486,6 +486,14 @@ - (NSUInteger)bleachMentionsWithinRange:(NSRange)bleachRange {
                                  for (NSString *key in selectedAttributes) {
                                      [buffer removeAttribute:key range:HKW_FULL_RANGE(input)];
                                  }
+                                 UIFont *parentFont = parentTextView.fontSetByApp;
+                                 UIColor *parentColor = parentTextView.textColorSetByApp;
+                                 if (parentFont) {
+                                     [buffer addAttribute:NSFontAttributeName value:parentFont range:HKW_FULL_RANGE(input)];
+                                 }
+                                 if (parentColor) {
+                                     [buffer addAttribute:NSForegroundColorAttributeName value:parentColor range:HKW_FULL_RANGE(input)];
+                                 }
```

Comment:
can we not just call`stripCustomAttributesFromTypingAttributes`?

also can we put the bleemention transformation into a helper to avoid duplicate code?

Reply:
can't do strip, because that's typing attributes - for future typing. Not for restoring attributes to an existing mention

and yes moved to helper